src/containers/Settings.jsx

```diff
@@ -323,7 +327,7 @@ class Settings extends React.Component {
                   {formatTextingHours(organization.textingHoursEnd)}
                 </span>
                 {window.TZ
-                  ? ` in your organisations local time. Timezone ${window.TZ}`
+                  ? ` in your organisation's local time. Timezone ${window.TZ}`
```

Comment:
Not sure if we would rather write `organization's` instead of `organisation's`, but that's just a US vs UK/Australia thing!